content/community/video_miner.md

```diff
@@ -0,0 +1,12 @@
+---
+publish: true
+image: >-
+  https://user-images.githubusercontent.com/16746274/222543634-6ffa38ad-d692-4dee-b800-1d969ad6dec7.png
```

Comment:
Please upload the attached logo to cloudinary and update the image url.
![videominer_logo_ecosystem2-01-02](https://user-images.githubusercontent.com/16746274/222544353-2bf8b79d-e9e8-4102-854a-83827149820e.png)